frameworks/angular/workspace/src/app/services/notifications.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from "@angular/core";
+import { create, type NotificationOptions, register } from "@openfin/workspace/notifications";
+import { from } from "rxjs";
+import { type PlatformSettings } from "./types";
+
+/**
+ * Note openfin-notifications package is deprecated - use @openfin/workspace/notifications (as shown here)
+ */
+@Injectable({ providedIn: "root" })
+export class NotificationsService {
+	register(platformSettings: PlatformSettings) {
+		console.log("Registering the Notifications provider");
+		return from(
+			register({
+				notificationsPlatformOptions: platformSettings,
+			}),
+		);
+	}
+
+	create(config: NotificationOptions): void {
+		create({
+			platform: fin.me.identity.uuid,
```

Comment:
When specifying a platform for a notification it should match the id passed to the notification register function.